### PR TITLE
Linux script for create icns

### PR DESCRIPTION
**Pull request by Fidius-jko**
#### Resizing icons and creating an icns file in Linux!

### CODE REVIEW

File: build/macos/create_icns_linux.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+rm -rf AppIcon.iconset/*
+mkdir -p AppIcon.iconset
+magick ../icon_1024x1024.png -resize 16x16    AppIcon.iconset/icon_16x16.png
```
Comment (NiklasEi) [inline build/macos/create_icns_linux.sh:5]: I need to use `convert` here instead of `magick`

File: build/macos/create_icns_linux.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+magick ../icon_1024x1024.png -resize 32x32    AppIcon.iconset/icon_16x16@2x.png
+magick ../icon_1024x1024.png -resize 32x32    AppIcon.iconset/icon_32x32.png
+magick ../icon_1024x1024.png -resize 64x64    AppIcon.iconset/icon_32x32@2x.png
+magick ../icon_1024x1024.png -resize 128x128  AppIcon.iconset/icon_128x128.png
+magick ../icon_1024x1024.png -resize 256x256  AppIcon.iconset/icon_128x128@2x.png
+magick ../icon_1024x1024.png -resize 256x256  AppIcon.iconset/icon_256x256.png
+magick ../icon_1024x1024.png -resize 512x512  AppIcon.iconset/icon_256x256@2x.png
+magick ../icon_1024x1024.png -resize 512x512  AppIcon.iconset/icon_512x512.png
+cp ../icon_1024x1024.png AppIcon.iconset/icon_512x512@2x.png
+png2icns ./AppIcon.icns AppIcon.iconset/icon_16x16.png AppIcon.iconset/icon_32x32.png AppIcon.iconset/icon_128x128.png AppIcon.iconset/icon_256x256.png AppIcon.iconset/icon_512x512.png
```
Comment (NiklasEi) [inline build/macos/create_icns_linux.sh:15]: Will this automatically use all the `@2x` files?

**Review by NiklasEi — CHANGES_REQUESTED** [review]
Thanks, this will be a big help for all users without a mac.

Can you mark the script as executable, please?

**Comment by Fidius-jko** [comment]
> Can you mark the script as executable, please?

The file appears to be executable.
I don't understand how to fix this

**Comment by NiklasEi** [comment]
> > Can you mark the script as executable, please?
> 
> The file appears to be executable. I don't understand how to fix this

I had to run `chmod +x create_icns_linux.sh` to be able to execute it and that command created a git diff. But if it is already executable on your end we can just merge this and I can verify later.

**Comment by Fidius-jko** [comment]
I understood why it works for me. Because my folder is on a partition formatted in NTFS. I fixed this.

**Review by NiklasEi — APPROVED** [review]
Thank you :)

**Pull request merged**